administrator/components/com_templates/src/Model/TemplateModel.php

```diff
@@ -1177,9 +1177,9 @@ public function createOverride($override)
             $name         = end($explodeArray);
             $client       = ApplicationHelper::getClientInfo($template->client_id);
 
-            if (stristr($name, 'mod_') != false) {
+            if (stristr($name, 'mod_')) {
```

Comment:
Better use strict comparison with false. I even want to change it to use str_contains, but that would be for a different PR
```suggestion
            if (stristr($name, 'mod_') !== false) {
```